schema_test.go

```diff
@@ -337,21 +337,21 @@ func (s *S) TestFloat(c *gc.C) {
 	c.Assert(err, gc.IsNil)
 	c.Assert(out, gc.Equals, float64(1))
 
-	out, err = sch.Coerce(int64( math.MaxInt64 ), aPath)
+	out, err = sch.Coerce(int64(math.MaxInt64), aPath)
```

Comment:
Ugh, who puts spaces inside parens?!

Reply:
I could look at blame, but I think it is old.

Reply:
The same people that put 2 spaces after a full stop?